.github/workflows/ci.yml

```diff
@@ -30,23 +36,23 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Install Node.js ${{ matrix.arch }}
-        uses: niik/setup-node@f9547c97f4c519f71dc42e652d6d2c53f9527c1d
```

Comment:
Didn't notice we used a custom `setup-node` 🤯

Reply:
Yeah, and from some super untrustworthy source too!